doc/nrf/protocols/matter/getting_started/tools.rst

```diff
@@ -164,132 +164,119 @@ For more information about the ZAP tool, see the official `ZCL Advanced Platform
 Installing the ZAP tool
 =======================
 
-To install the ZAP tool using the recommended method, complete the following steps:
+.. tabs::
 
-1. Download the ZAP package containing pre-compiled executables and libraries and extract it:
+   .. tab:: Automatic Installation Using the Matter west Command
 
-   a. Open your installation directory for the |NCS| in a command line.
-   #. Navigate to :file:`modules/lib/matter`.
-   #. Run the helper script to download and extract the ZAP package:
+      You can use the Matter West command to automatically download and install the appropriate ZAP tool version dedicated to your current Matter revision.
```

Comment:
It would be worth explaining that both `zap-gui` and `zap-generate` commands automatically install the right version of the ZAP tool in the `.zap-install` directory in the Matter repository within NCS. Then, you can use `zap-gui` as an example like you did.